HACKING.md

```diff
@@ -87,7 +87,7 @@ The two commands above run all the available tests on all versions of Python sup
 
 The first time you run `make`, you may be asked for your password. This is in order to install `pandoc`, a tool used for processing the README file.
 
-Running all tests requires that you have all the supported versions of Python installed. These are 2.6, 2.7, 3.2, 3.3, 3.4 and 3.5. Try to use your package manager to install them if possible. Otherwise, go to [python.org/downloads][] and install them manually.
+Running all tests requires that you have all the supported versions of Python installed. These are 3.7 thru 3.12. Try to use your package manager to install them if possible. Otherwise, go to [python.org/downloads][] and install them manually.
```

Comment:
Some of the wording here is out of date because now we use `pyenv` in the Makefile. But I wanted to keep the scope of the PR minimal and just change references to the Python versions.